templates/openstack/form-data.json

```diff
@@ -0,0 +1,51 @@
+{
+    "form": {
+      "/openstack": {
+        "templatePath": "/openstack/index.html",
+        "isModal": true,
+        "modalId": "openstack-modal",
+        "formData": {
+          "title": "Get in touch with the OpenStack team",
+          "introText": "Fill in this form and a member of our team will be in touch shortly.",
+          "formId": "6024",
+          "returnUrl": "/openstack#contact-form-success",
+          "product": "openstack"
+        },
+        "fieldsets": [
+          {
+            "title": "What advice are you looking for?",
```

Comment:
The required fields are [different](https://na-sjg.marketodesigner.com/m#FOP6024) on marketo, I think that might be causing some failed submissions

Reply:
Gave a quick test by not including the un-required field and my submission still goes through to marketo